contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,59 +1,140 @@
 category: base
 doc: |
-  Description of peaks, their functional form or measured support.
+  Base class for describing spectral peaks, their functional form, and support values
+  (i.e., the discretization (points) at which the function has been evaluated).
 symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
   n_support: |
-    Number of support points
+    Number of support points.
 type: group
 NXpeak(NXobject):
   label:
     doc: |
-      Human-readable identifier to specify which concept/entity
+      Human-readable label which specifies which concept/entity
       the peak represents/identifies.
-  peak_model:
+  lineshape:
     doc: |
       Is the peak described analytically via a functional form
       or is it empirically defined via measured/reported
-      intensity/counts as a function of an independent variable.
+      intensity/counts as a function of an independent variable?
       
-      If the functional form is not empirical or gaussian, users
-      should enter other for the peak_model and add relevant details
+      If the functional form is not empirical or Gaussians, users
+      should enter other for the description and add relevant details
       in the NXcollection.
-    enumeration: [empirical, gaussian, lorentzian, other]
+    enumeration: [empirical, gaussian, lorentzian, voigt, other]
```

Comment:
If you limit this here, provide formulas for the enumerated peak shapes, plus expected parameters and their meaning.

Reply:
Not limited anymore.